SCons/Platform/__init__.py

```diff
@@ -54,6 +54,10 @@
 import SCons.Subst
 import SCons.Tool
 
+# Gets set to True in Main._main() when the '--process-spawner'
+# option was given on the command-line for speeding up process spawning.
+process_spawner = False
```

Comment:
Not sure if this is the right spot to place it? It's POSIX only so it should be in `posix.py` maybe? I followed #3703 on this one.